src/test/java/de/wejago/mqtt2influx/service/MqttSubscriberServiceTest.java

```diff
@@ -22,34 +24,33 @@
 
 @ExtendWith({OutputCaptureExtension.class, MockitoExtension.class})
 class MqttSubscriberServiceTest {
-    private final String TOPIC_TEST = "topic/test";
     @Mock
     private IMqttClient mqttClient;
-
     @Mock
     private MqttConnectOptions mqttConnectOptions;
-
     @Mock
-    private SensorMessageSubscriber sensorMessageSubscriber;
-
+    private DevicesConfig devicesConfig;
```

Comment:
No need to mock that

Reply:
I agree it would be better without mock. But how can I do it without mocking it? `devicesConfig` is a property of the `MqttSubscriberService`.